CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.3)
+cmake_minimum_required(VERSION 3.5)
```

Comment:
Without this there can be warnings that read "Compatibility with CMake < 3.5 will be removed from a future version of CMake"